README.md

```diff
@@ -1,5 +1,5 @@
 # hebi_gazebo
-
+### NOTE: Intended for use with Gazebo versions 7 (Ubuntu 16.04) and 9 (Ubuntu 18.04) only, other configurations untested
 This repository holds the plugins and configuration files used to simulate HEBI robots in Gazebo.
 In addition, you will need the model files contained in the hebi_description package, or your own hrdf/urdf in the case of a custom robot.
```

Comment:
(line 4-5) Can you hyperlink "hebi_description" and "hebi_moveit_configs" to their relevant repos so that people can easily go there and get these dependencies?

(line 7) Should be "These instructions" (add the s). Missing a full stop at the end. 

(line 10) Put a colon at the end.

(line 27) Consider making this note bolded? It's important enough to warrant more attention i think.

(line 30) Maybe hyperlink "hebi_description" once again.

(line 38 - 42)
![image](https://user-images.githubusercontent.com/25698831/70267814-32d64a00-176d-11ea-814c-2d442cf8e0fe.png)
Is there any way to actually make the (a) and (b) points appear as proper bullet points?
I think point (a) is also missing words between "terminal" and "gazebo"?

(line 44) It should be "through standard" (take out "the")